README.md

```diff
@@ -34,11 +34,11 @@ We are still using node 16.x and npm 8.x. If you are [using nvm](https://github.
 - `npm run e2e`: Runs the Playwright end-to-end tests locally.
 - `npm run e2e:headed`: Runs end-to-end tests in headed mode debug mode. Useful if you need to debug why a particular test isn't work. For example, to debug the `table.spec.ts` test directly, you could run `npm run e2e:headed -- ./tests/table.spec.ts`.
 
-Edit `.env.local` in each package to contain the following pointing to your local DHC address. These are needed for the session websocket and for things like notebooks to be proxied correctly by Vite.
+If your DHC address is different from the default `http://localhost:10000`, edit `.env.local` in each package to contain the following pointing to your local DHC. These are needed for the session websocket and for things like notebooks to be proxied correctly by Vite.
```

Comment:
This needs to be `.env.development.local` if running locally. The load priority of the files is `.env.{mode}.local`, `.env.{mode}`, `.env.local`, `.env`

So if running in dev mode, the proxy URL won't be overwritten

We should probably just move `VITE_PROXY_URL` into `.env` though. If running `npm run preview` after a build, Vite starts the preview server in production mode. So the proxy URL isn't set and Vite won't proxy the requests

Reply:
Moved `VITE_PROXY_URL` to `.env`.

Now `.env.local` is the correct location for the override if we want it to work for both preview and dev, right?

Reply:
Yes, `.env.local` is the right spot and it will work for both